app/src/main/java/com/asfoundation/wallet/App.java

```diff
@@ -104,17 +107,26 @@ private void initializeRakam() {
       Log.e(TAG, "error: ", e);
     }
 
+    int userLevel = idsRepository.getGamificationLevel();
```

Comment:
Since we are setting so much info to initialise the Rakam, should we use rx to make it asynchronous?